Tools/vagrant/initvagrant-desktop.sh

```diff
@@ -14,7 +14,7 @@ fi
 
 apt-get update
 
-apt-get install -y ubuntu-desktop
+apt-get install -y ubuntu-desktop dbus-x11
```

Comment:
Please test a vagrant-up of our oldest supported LTS release to make sure this doesn't kill it - or adjust the patch so it is only installed under jammy.